ios/Classes/API/UsercentricsBannerProxy.swift

```diff
@@ -26,11 +26,26 @@ struct UsercentricsBannerProxy: UsercentricsBannerProxyProtocol {
 
     func showSecondLayer(bannerSettings: BannerSettings?,
                          completionHandler: @escaping (UsercentricsConsentUserResponse) -> Void) {
-        guard let flutterVC = UIApplication.shared.delegate?.window??.rootViewController as? FlutterViewController else { return }
+        guard let flutterVC = getFlutterViewController() else { return }
 
         UsercentricsBanner(bannerSettings:bannerSettings).showSecondLayer(hostView: flutterVC) { response in
             completionHandler(response)
         }
     }
 
+    func getFlutterViewController() -> FlutterViewController? {
```

Comment:
@alibasta Do you think this solution covers all the edge cases?
Thanks! 🙏

Reply:
Thanks!! Works like a charm! 🎉